src/ui/components/search/searchcomponent.js

```diff
@@ -524,10 +574,12 @@ export default class SearchComponent extends Component {
       }
     }
 
-    inputEl.blur();
-    DOM.query(this._container, '.js-yext-submit').blur();
-    if (this.isUsingYextAnimatedIcon) {
-      this.animateIconToYext();
+    if (!this._showLoadingIndicator) {
+      inputEl.blur();
```

Comment:
is it ok to move inputEl.blur() into this if statement?

Actually for fun I went digging and this originally came from https://github.com/yext/answers-search-ui/pull/174/files
I wonder if this was related to the weird mobile keyboard techops we had a while back that Connor looked into

Reply:
ah ok I see. yeah it would bring back that techop issue. 
I made an update. `inputEl.blur()` gets call without that if statement (got rid of the call `animateIconToYext ()` that is right after this, since the blur will trigger the `focusout` listener to animate it anyway). But, with this change, the call to `window.requestAnimationFrame(..)` will not get register each time since the time frame between each call is too quick for the function to set `this.isRequestingAnimationFrame` back to false. This mean we can't rely on hiding just the previous iconState. I had to change the code back to hiding all icons with `.js-yxt-SearchBar-Icon` instead of just the prevState.

Reply:
Now that I'm thinking about it, the original thought was that pressing enter while typing on the searchbar should still remain focus on the searchbar. But if this lead to weird mobile keyboard, then with this update to always blur on submit, technically we will never have a scenario where we go from loading state to magnifying glass..darn. I will remove the product request for the static magnifying glass then.